docs/src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import React from 'react';
+import { cssModules } from 'bpk-react-utils';
+import BpkContentContainer from 'bpk-component-content-container';
+import Helmet from 'react-helmet';
+
+import BpkMarkdownRenderer from '../DocsPageBuilder/BpkMarkdownRenderer';
+import Heading from '../Heading';
+import IntroBlurb from '../IntroBlurb';
+
+import STYLES from './MarkdownPage.scss';
+
+const getClassName = cssModules(STYLES);
+
+type Props = {
+  title: string,
+  subtitle: ?string,
```

Comment:
I don't think this should be called `subtitle`, because it's a paragraph

Reply:
There's nothing that says subtitles are a single line. This usage meets the dictionary definition almost perfectly:

<img width="756" alt="Screen Shot 2020-10-26 at 16 59 32" src="https://user-images.githubusercontent.com/73652/97203386-a1206280-17ac-11eb-91f2-ea4bd36c5880.png">

Reply:
Actually that makes sense 😂

Reply:
You know me and grammar 😛